updated R tech guides url, R version in readme

GuidoMaggio opened the pull request:
- updated R tech guides url 
- updated R version in readme (4.2.3)

GuidoMaggio committed:
updated R tech guides url, added target = blank, updated R version in readme

RolandASc commented:
@GuidoMaggio fine with the changes, but the build is broken meanwhile (because of tinytex installation it seems)

riccardoporreca commented on README.md:
> added target = blank to external sites

@GuidoMaggio, there is no support for `{target="_blank"}` in GitHub's README's, see https://github.com/miraisolutions/SmaRP/tree/a45216fbf361ff0cc5e138565bf1d6951cb24b7f#readme

Please revert the change and update PR  title/description

GuidoMaggio committed:
reverted target blank

RolandASc commented:
@GuidoMaggio indeed, the `tinytex` installation just isn't stable. We would probably have to update the whole thing (base image `FROM rocker/r-ver:4.2.3` and *CTAN_REPO*) again.

RolandASc committed:
separate commands

expecting some package installation error on the first command, testing if this helps

RolandASc commented:
Well ok, maybe my quick fix attempt was good enough 🤨

riccardoporreca commented:
> @GuidoMaggio indeed, the `tinytex` installation just isn't stable. We would probably have to update the whole thing (base image `FROM rocker/r-ver:4.2.3` and _CTAN_REPO_) again.

I believe the issue is that <https://github.com/rstudio/tinytex/raw/main/tools/install-unx.sh> is not versioned, and even worse the set of packages it installs from <https://yihui.org/gh/tinytex/tools/pkgs-custom.txt>, so using an old, version-stable `CTAN_REPO` might not be enough.

Not relevant for this PR though, we don't plan to redeploy in any case, so even if the built image is not fully functional it is not a big deal.

RolandASc commented:
> I believe the issue is that https://github.com/rstudio/tinytex/raw/main/tools/install-unx.sh is not versioned, and even worse the set of packages it installs from https://yihui.org/gh/tinytex/tools/pkgs-custom.txt, so using an old, version-stable `CTAN_REPO` might not be enough.

Exactly.  
Some of those things haven't been updated in 3 years. Actually there is `https://tinytex.yihui.org/` since long. But it does seem that an extra new Latex package missing in the old repository just produces a local error while the rest is installed properly.

We can also try to switch the image once in the Azure deployment, it is a couple of clicks, and switch it back after some testing.
